packages/back-end/src/controllers/organizations.ts

```diff
@@ -587,6 +591,10 @@ export async function postInviteAccept(req: AuthRequest, res: Response) {
       throw new Error("Must be logged in");
     }
     const org = await acceptInvite(key, req.userId);
+
+    if (!org) {
+      throw new Error("You can't accept an invite if you're already a user");
+    }
```

Comment:
This is the error that an admin would see if they clicked on an invite link meant for someone else. This will block the admin from accidentally adding themselves to organization.members a second time causing duplicates. Open to suggestions on the copy. For now, I'm moving it inside of `services/organizations.ts` based on feedback below.